standalone_self_upgrade.cf.in

```diff
@@ -764,6 +780,13 @@ body common control
 {
       version => "CFEngine Standalone Self Upgrade @VERSION@";
 
+
+    control_common_tls_min_version_defined::
```

Comment:
maybe a "see controls/cf_serverd.cf" in here and in update.cf?

Reply:
I added cross comments about the other body to find related settings.